### PR TITLE
chore: remove aniyomiorg/aniyomi-extensions

### PR DESCRIPTION
**Pull request by Secozzi**
Same rationale as removing the official manga extensions, people are just going to end up using extensions they don't know how to/shouldn't use, and i can confirm at least one extension will be maintained elsewhere.

Note: ISSUE_TEMPLATE.md is removed in #1601 so it isn't changed here

Changing the basline after its made is probably not a good idea, but this pr was made before detekt was added so whatever

### CODE REVIEW

**Comment by hollowshiroyuki** [comment]
so you confirm you'll maintain jellyfish elsewhere ? provide link at least 💢

File: .github/ISSUE_TEMPLATE/report_issue.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ body:
           required: true
         - label: I have written a short but informative title.
           required: true
-        - label: If this is an issue with an official anime extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
+        - label: If this is an issue with an anime extension, I should be opening an issue in the extensions repository.
```
Comment (abdallahmehiz) [inline .github/ISSUE_TEMPLATE/report_issue.yml:96]: It should be "extension's"

File: .github/ISSUE_TEMPLATE/request_feature.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ body:
           required: true
         - label: I have written a short but informative title.
           required: true
-        - label: If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
+        - label: If this is an issue with an extension, I should be opening an issue in the extensions repository.
```
Comment (abdallahmehiz) [inline .github/ISSUE_TEMPLATE/request_feature.yml:33]: Here too

**Review by abdallahmehiz — CHANGES_REQUESTED** [review]
Sorry, secco

**Review by abdallahmehiz — DISMISSED** [review]

**Review by abdallahmehiz — APPROVED** [review]

**Comment by hollowshiroyuki** [comment]
rest in piss you won't be missed

**Pull request closed without merging**